js/todos.js

```diff
@@ -1,37 +1,27 @@
-var App = new (Backbone.View.extend({
+// Initialize global object
+let App = {
     Models: {},
     Views: {},
     Collections: {},
-    template: _.template(
-        '<div class="main-container">' +
-        '<div class="todo-list-container"><h1>ToDo List</h1>' +
-        '<div id="app"></div>' +
-        '</div></div>' 
-    ),
+};
 
-    render: function () {
-        this.$el.html(this.template());
-        return this;
-    },
-    start: function () {
-        this.$el.append(this.render().el);
+$(document).ready(function () {    
```

Comment:
Shouldn't be necessary with `async` and `defer` or if you move script tags to before the closing body tag.